File: c-sharp/ws-client/model/parameters/OrderBookParams.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ws_client.model.parameters
{
    class OrderBookParams : Params
    {
        public OrderBookParams(string symbol, int? depth) : base(ChannelTypes.ORDERBOOK, symbol)
        {
            this.depth = depth;
        }

        public int? depth { get; set; }
    }
}

File: java/src/main/java/model/params/OrderBookParams.java
package model.params;

import model.ChannelTypes;

public class OrderBookParams extends Params {

    private Integer depth;

    public OrderBookParams(String symbol, Integer depth) {
        super(ChannelTypes.ORDERBOOK, symbol);
        this.depth = depth;
    }

    public Integer getDepth() {
        return depth;
    }

    public void setDepth(Integer depth) {
        this.depth = depth;
    }

    @Override
    public String toString() {
        return "OrderBookParams{" +
                super.toString() +
                ", depth=" + depth +
                "} ";
    }
}
